Kitodo/src/main/java/org/kitodo/production/editor/XMLEditor.java

```diff
@@ -114,12 +114,10 @@ public String getCurrentConfigurationFile() {
      *            name of the configuration to be loaded
      */
     public void loadXMLConfiguration(String configurationFile) {
-        try (StringWriter stringWriter = new StringWriter()) {
+        try {
             currentConfigurationFile = configurationFile;
             this.configurationFile = KitodoConfigFile.getByName(configurationFile);
-            XMLConfiguration currentConfiguration = new XMLConfiguration(this.configurationFile.getAbsolutePath());
-            currentConfiguration.save(stringWriter);
-            this.xmlConfigurationString = stringWriter.toString();
+            this.xmlConfigurationString = Files.lines(this.configurationFile.getFile().toPath()).collect(Collectors.joining("\n"));
```

Comment:
`Files.lines` returns a `Stream<String>`, which isn't closed here in case of an `IOException`, if I am not mistaken. Would it be possible to use a `try-with-resources` statement here to make sure the stream is always closed?